test/common/wsServerWorker.js

```diff
@@ -0,0 +1,30 @@
+
+
+var sinon = require('sinon');
+var WAMPServer = require('wamp-socket-cluster/WAMPServer');
+var testConfig = require('../config.json');
+
+var necessaryRPCEndpoints = {
```

Comment:
Variable duplicated with `wsServer`

Reply:
Since later we will refactor and remove `wsServer` file completely and move to `wsServerMaster`. So didn't introduced a new dependency here.

Reply:
Ok, please log the issue then.

Reply:
Done. https://github.com/LiskHQ/lisk/issues/950